src/App.tsx

```diff
@@ -1,41 +1,63 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './App.scss';
 import './styles/general.scss';
 import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
+import { getAllUsers } from './api/api';
+import { User } from './react-app-env';
 
-const App: React.FC = () => (
-  <div className="App">
-    <header className="App__header">
-      <label>
-        Select a user: &nbsp;
+const App: React.FC = () => {
```

Comment:
Don't use `React.FC` as it always inserts `children` type to your props

```suggestion
const App = () => {
```